docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,245 @@
+
+## Application and Viewer
+
+[`NUIWidgetApplication`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) makes diverse class instances whenever [`WidgetView`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.WidgetView.html) requests for a widget instance.
+
+The widget instance has its own life-cycle similar to that of the widget application. However, the widget instance is an object created by the widget viewer application. Many widget instances can be running on a widget application process.
+
+> [!NOTE]
+> The case to use many widget instances in one widget application is known as multi-instance. In some devices, the multi-instance may not be supported. If a device does not support multi-instance, an error message is displayed.
+
+[`WidgetView`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.WidgetView.html) shows the contents drawn by [`Widget`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Widget.html) on the screen.
+
+To summarize, you [create a NUI WidgetApplication](#create-a-nui-widget-application), [create a NUI WidgetView](#create-a-nui-widget-view), and use Widget to view layout on the screen.
+
+## Instance States and Events
+The following link illustrates the widget instance states during the instance life-cycle:
+https://docs.tizen.org/application/dotnet/guides/applications/widget-app#instance
+
+The `Widget` class provides interface for creating custom widget.
+The following table lists the main functions to manage the widget instance:
+
+**Table: Main functions**
+| Function           | Description                                                                                   |
+|--------------------|-----------------------------------------------------------------------------------------------|
+| `OnCreate()`       | This function is called after the widget instance is created.                                 |
+| `OnTerminate()`    | This function is called after the widget instance is terminated.                              |
+| `OnPause()`        | This function is called when the widget is invisible.                                         |
+| `OnResume()`       | This function is called when the widget is visible.                                           |
+| `OnResize()`       | This function is called before the widget size is changed.                                    |
+| `OnUpdate()`       | This function is called when an event for updating the widget is received.                    |
+| `SetContentInfo()` | This function sends data about current state of widget instance to a widget viewer application. If you want to save the current state of widget before deleting it, you can save it to the current status using `SetContentInfo()`.|
```

Comment:
@tscholb,

Please respond to the query:

https://github.com/Samsung/tizen-docs/pull/1052#discussion_r400615349

Reply:
We got confirmation from @tscholb about this update.

https://github.com/Samsung/tizen-docs/pull/1052#discussion_r441972273

> i thought"save" is better, so i changed it.

Thank you both for checking and confirmation.